packages/react/src/components/Forms/Input/index.scss

```
@@ -1,23 +1,116 @@
-.c__input {
-  height: 2rem;
-  padding: 0 14px;
-  box-sizing: border-box;
-  border-radius: 2px;
-  border: 1px var(--c--theme--colors--greyscale-300) solid;
+.c__input__wrapper {
+
+  @mixin text-style {
```

A placeholder fits well here.

```suggestion
  %text-style {
```